Automattic/vip-cli
# Run terminate handler when command is finished or cancelled

## Description
When a command is prematurely terminated (via CTRL-C), we need to run cleanup tasks such as notifying the command runner to stop the commands.

## Steps to Test
1. Pull down PR and related API PR
2. Run `wp user create --prompt` and wait for the first prompt
3. Ctrl-C to exit
4. Verify it was cancelled (in the app dashboard)
5. Repeat for `wp shell`
6. Repeat for the psuedo shell
7. Try a few "long-running" commands (may be difficult to hit the timing on small sites) `wp option list` `wp jetpack sync start`

Note that for commands with a lot of output (`wp option list` for example), the command may still stream output text for a second or two after it completes. So if you cancel it at that point, it will show as 'completed' since it did actually finish running.

Closes #416
